controllers/numaresourcesoperator_controller.go

```diff
@@ -319,6 +320,15 @@ func (r *NUMAResourcesOperatorReconciler) syncNUMAResourcesOperatorResources(ctx
 		return daemonSetsNName, err
 	}
 
+	// ConfigMap should be provided by the kubeletconfig reconciliation loop
+	if r.RTEManifests.ConfigMap != nil {
+		cmHash, err := hash.ComputeCurrentConfigMap(ctx, r.Client, r.RTEManifests.ConfigMap)
+		if err != nil {
+			return daemonSetsNName, err
+		}
+		rtestate.UpdateDaemonSetHashAnnotation(r.RTEManifests.DaemonSet, cmHash)
```

Comment:
in the future (ofc unrelated to this PR) we will need to move `Update*` utilities in their own package. Perhaps like
```
pkg/manifests/rte
pkg/manifests/sched
```
mimiching (and extending) what `deployer` does